yesod-test/yesod-test.cabal

```diff
@@ -1,5 +1,5 @@
 name:               yesod-test
-version:            1.5.0.1
+version:            1.6.0.0
```

Comment:
Oh so it does. I'd managed that on accident, thinking I'd changed the type signature of something that _was_ exported. Nuts. I'll fix it, sorry!